mlflow/utils/promptlab_utils.py

```diff
@@ -0,0 +1,143 @@
+import json
+from datetime import datetime
+from typing import List
+
+from mlflow.entities import Param
+
+
+def create_conda_yaml_file(mlflow_version: str) -> str:
+    conda_yaml = f"""
+    name: mlflow-env
+    channels:
+      - defaults
+    dependencies:
+      - pip
+      - pip:
+        - mlflow[gateway]=={mlflow_version}
+    """
```

Comment:
/ai In this python code, I think we can use `textwrap.dedent` to remove trailing space on each line. Is it slow?